crowdstrike/table_crowdstrike_user_role.go

```diff
@@ -0,0 +1,104 @@
+package crowdstrike
+
+import (
+	"context"
+
+	"github.com/crowdstrike/gofalcon/falcon"
+	"github.com/crowdstrike/gofalcon/falcon/client/user_management"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func tableCrowdStrikeUserRole(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "crowdstrike_user_role",
+		Description: "TODO.",
+		List: &plugin.ListConfig{
+			Hydrate: listCrowdStrikeUserRole,
+		},
+		Get: &plugin.GetConfig{
+			Hydrate:    getCrowdStrikeUserRole,
+			KeyColumns: plugin.AllColumns([]string{"id"}),
+		},
+		Columns: []*plugin.Column{
+			{Name: "id", Description: "TODO", Type: proto.ColumnType_STRING, Hydrate: getCrowdStrikeUserRole},
```

Comment:
This table feels a bit light, is there any other info we can get about a user role?

Reply:
Not really! The SDK doesn't offer much in the way of roles.